docs/shuttle-procedure.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Product guidelines
+tags: docs
+---
+
+# Considerations during Shuttle Procedure
+
+This article defines a list of steps to follow during the [Shuttle Procedure](http://atenea.marfeel.com/display/PRODUCT/How+to+shuttle+Gutenberg) in order to **prevent** and **minimize** the impact on clients.
```

Comment:
we have this in MarfeelDocs already - once we know where we put this article we have to change the link

Reply:
I just saw it.  Should go in Continuous Deployment?

Reply:
Mmh, concept-wise, it could go there too yes!

But it's quite different from the "project" of the DX team that is currently documented there :/